### PR TITLE
Testing cd

### PR DESCRIPTION
**Pull request by chadrick-d-dev**
## Description
- wrote tests for App.js
- Attempted to write testing for DailyReading.js need to have it changed to a class component. Testing is commented out/work in progress/ for when we change DailyReading to a class component.

## Motivation and Context
- Needed to test App.js and DailyReading.js

## How Has This Been Tested?
- Performed unit testing, including an async test in App.js.
- In DailyReading I wanted to mock the card passed through props. I was unable to successfully mock the card.

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

## Issues
- Need to understand how or if props can be mocked within React testing.

### CODE REVIEW

File: src/App/App.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ import { fetchCards } from '../fetch/fetch-requests';
 
 class App extends Component {
   state = { 
-
+    deck: undefined
```
Comment (CalebCyphers) [inline src/App/App.js:12]: Good addition here

File: src/App/App.test.js [hunk 2]
```diff
@@ -1,8 +1,38 @@
...
+      {
+        "id": 1,
+        "reversed": true,
+        "numeral": "O",
+        "title": "The Bard",
+        "image": "https://i.imgur.com/QBKZRaT.jpg",
+        "theme": "Recklessness",
+        "description": "Now is the time to start something new, even if you are unsure of the outcome. Trust in your performance, but also take time to make sure that you are well prepared."
+      }
+    ])
```
Comment (CalebCyphers) [inline src/App/App.test.js:29]: I can help with the mocking here

File: src/DailyReading/DailyReading.test.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { render, screen, waitForgit } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import DailyReading from './DailyReading';
+
+// describe('DailyReading', () => {
```
Comment (CalebCyphers) [inline src/DailyReading/DailyReading.test.js:6]: Thanks for working on this, we'll get it running no worries!

**Review by CalebCyphers — APPROVED** [review]

**Review by kathrynljackson — COMMENTED** [review]
Looks like we have a solid testing foundation!

**Pull request merged**